examples/nuxt-app/app.vue

```diff
@@ -20,6 +20,8 @@
         </MtText>
 
         <SwTextField label="A text field" v-model="text" />
+
+        <MtSearch />
```

Comment:
I think just making the `mt-search` component public is enough. We don't need to use every single component in the nuxt example app.

Reply:
`mt-search` does depend on the `$t` which is provided [here](https://github.com/shopware/meteor/pull/248/files#diff-12ee38794478c53d0ef713c4a4c11b9587d3afe5f9cc1f8216d63bf5a01da0ac). Though this isn't correct solution (using correct Vue/Nuxt i18n library would be the right approach), we do have an "example app" where we can show that the component actually depends on the i18n library.